mappings/net/minecraft/util/math/Direction.mapping

```diff
@@ -64,9 +72,11 @@ CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
 	METHOD method_23224 getRotationQuaternion ()Lnet/minecraft/class_1158;
 	METHOD method_23225 transform (Lnet/minecraft/class_1159;Lnet/minecraft/class_2350;)Lnet/minecraft/class_2350;
 	METHOD method_23955 getUnitVector ()Lnet/minecraft/class_1160;
+	METHOD values ()[Lnet/minecraft/class_2350;
```

Comment:
```suggestion
```
useless entry i think

Reply:
Yeah, I think I did that one by accident